helm/openwhisk/values.yaml

```diff
@@ -249,20 +255,69 @@ nginx:
 # Controller configurations
 controller:
   imageName: "openwhisk/controller"
-  imageTag: "c5970a6"
+  imageTag: "a1639f0"
+  imagePullPolicy: "IfNotPresent"
+  replicaCount: 1
+  restartPolicy: "Always"
+  port: 8080
+  options: ""
+  jvmHeapMB: "1024"
+  jvmOptions: ""
+  loglevel: "INFO"
+
+# Scheduler configurations
+scheduler:
+  enable: true
```

Comment:
Do we want this to default to true or not? I set it to true here so that the travis tests would run by default with the scheduler, but perhaps the correct thing to do is run travis tests both with and without the scheduler?

Reply:
+1 to David's opinion.

We may take the same approach with the core repo.
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/3447251/190398762-83320684-d6a0-4c2d-a5c1-bb732b5e1643.png">

It has a dedicated CI pipeline for the new scheduler.

Reply:
I've added a scheduler check to the travis test matrix